wordpress-mobile/WordPress-Android
# RI: Topic details - Convert TagHeaderView + FollowButton to Kotlin

Parent Issue #12318

This PR converts `ReaderTagHeaderView` + `ReaderFollowButton` to Kotlin as per https://github.com/wordpress-mobile/WordPress-Android/pull/12355.

#### To test
Kotlin conversion was done using AndroidStudio kotlin conversion tool.
Just check that file history is maintained and there's no use of double-bang (!!) operator.

#### Merge instruction
PR targets 15.4. It can be merged to 15.3 or 15.4. To be safe, let's wait for 15.3 code freeze and merge in 15.4.

PR submission checklist:
